packages/react-native-client/ios/RNFishjamClientModule.swift

```diff
@@ -86,6 +82,30 @@ public class RNFishjamClientModule: Module {
             return client
         }()
 
+        Name("RNFishjamClient")
+
+        Events(EmitableEvents.allEvents)
+
+        Property("peerStatus") {
+            rnFishjamClient.peerStatus.rawValue
+        }
+
+        Property("isMicrophoneOn") {
+            return rnFishjamClient.isMicrophoneOn
+        }
+
+        Property("isCameraOn") {
+            return rnFishjamClient.isCameraOn
+        }
+
+        Property("cameras") {
+            rnFishjamClient.getCaptureDevices()
+        }
+
+        Property("isScreenShareOn") {
+            return rnFishjamClient.isScreenShareOn
+        }
```

Comment:
**suggestion:** 
I know it is not your code, but if you refactored it lets be consistent and use return in every property.